examples/workshop/contract-example/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+#![no_std]
+
+use gstd::{msg, prelude::*, ProgramId};
+use gstd_meta::meta;
+
+meta! {
+    title: "GEAR Workshop Contract Example",
+    input: Vec<u8>,
+    output: Vec<u8>,
+    // Owner id
+    init_input: Vec<u8>,
+    init_output: Vec<u8>
+}
+
+#[derive(Debug)]
+struct State {
+    owner_id: Option<ProgramId>,
+}
+
+impl State {
+    fn set_owner_id(&mut self, owner_id: Option<ProgramId>) {
+        self.owner_id = owner_id;
+    }
+    fn owner_id(&mut self) -> Option<ProgramId> {
+        self.owner_id
+    }
+}
+
+static mut STATE: State = State { owner_id: None };
+
+#[no_mangle]
+pub unsafe extern "C" fn handle() {
```

Comment:
better use gstd-async entry point and it's decorator, then no need to handle reply manually